src/media/js/addon/containers/dashboardDetail.js

```diff
@@ -45,12 +45,12 @@ export class AddonDashboardDetail extends React.Component {
 
   renderDeleted() {
     return (
-      <div>
-        <h2>This Firefox OS add-on has been deleted.</h2>
-        <ReverseLink to="addon-dashboard">
-          Return to My Firefox OS Add-ons
-        </ReverseLink>
-      </div>
+      <PageSection>
+        <p>
+          This add-on has been deleted. <ReverseLink to="addon-dashboard">
```

Comment:
Yeah, the problem is that in this code:

    <p>
      Here is some text 
      <a href="">link</a> 
      and here is more text.
    </p>

JSX will render this:

> Here is some text.[link]()and here is more text

Stripping out any trailing spaces.